aws/components/apigateway/setup.ftl

```diff
@@ -303,27 +303,64 @@
             outputs={}
             dependencies=apiId
         /]
+
+        [#-- Throttling Configuration --]
+        [#local methodSettings = []]
+        [#local openapiIntegrations = 
+            getOccurrenceSettingValue(occurrence, [["apigw"], ["Integrations"]], true)]
+
+        [#if openapiIntegrations.Throttling?has_content]
```

Comment:
My bad. Looking at original code, we are always creating at least one methodSetting entry so this code doesn't need to be wrapped in the check for throttling.

Reply:
Updated: 15552a5